### PR TITLE
[17.0][MIG] purchase_blanket_order

### PR DESCRIPTION
**Pull request by ArnauCForgeFlow**
Standard Migration to 17.0

### CODE REVIEW

File: purchase_blanket_order/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "name": "Purchase Blanket Orders",
+    "category": "Purchase",
+    "license": "AGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "version": "17.0.1.0.0",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "summary": "Purchase Blanket Orders",
+    "depends": [
+        "purchase",
+        "web_action_conditionable",
```
Comment (LoisRForgeFlow) [inline purchase_blanket_order/__manifest__.py]: Looking at the module, this dependency looks like it is not needed. Can you check if it is really the case? If not, we probably can get rid of it during this migration.

Comment (ArnauCForgeFlow) [reply purchase_blanket_order/__manifest__.py]: You are right, I tested it and it looks like the module no longer needs this dependency. Removing it.

**Review by HviorForgeFlow — COMMENTED** [review]
I noticed some error when not configuring UOM settings and try to confirm a purchase blanket order.

image

**Comment by ArnauCForgeFlow** [comment]
@HviorForgeFlow solved!

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
The order confirmation message seems that is not parsed appropiately as html? 

image

**Comment by ArnauCForgeFlow** [comment]
> The order confirmation message seems that is not parsed appropiately as html?
> 
> image

Hey @LoisRForgeFlow 

The confirmation message is generated in purchase_request module, and there is an existing pr solving this problem: 
https://github.com/OCA/purchase-workflow/pull/2263

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks @ArnauCForgeFlow I tested the wrong module :rofl: 

This time I reviewed blanket order, functional review is good :+1:

**Review by HviorForgeFlow — APPROVED** [review]
Thanks for attending the comments.

**Pull request merged**